singer_sdk/exceptions.py

```diff
@@ -146,3 +146,14 @@ class InvalidJSONSchema(Exception):
 
 class InvalidRecord(Exception):
     """Raised when a stream record is invalid according to its declared schema."""
+
+    def __init__(self, error_message: str, record: dict) -> None:
+        """Initialize an InvalidRecord exception.
+
+        Args:
+            error_message: A message describing the error.
+            record: The invalid record.
+        """
+        super().__init__(f"Record Message Validation Error: {error_message}")
```

Comment:
Moving this here so all `BaseJSONSchemaValidator` implementations output the same base message and we don't need to adjust the test or repeat ourselves.